modules/claims_api/EVSS_CLAIMS.yml

```diff
@@ -56,8 +56,8 @@ info:
 tags:
   - name: claims
     description: EVSS claims status API
-servers:
-  - url: dev-api.va.gov/services/claims/{version}
+servers: 
+  - url: https://dev-api.vets.gov/services/claims/{version}
```

Comment:
Is the va.gov -> vets.gov change intentional?

Reply:
nope that was me messing up a merge.